src/components/widgets/GameNumberChart.tsx

```diff
@@ -0,0 +1,61 @@
+// @ts-nocheck
+
+import { onMount } from "solid-js"
+import { Chart, Title, Tooltip, Colors, TimeScale, type ChartOptions, type ChartData } from "chart.js"
+import { Bar } from "solid-chartjs"
+import "chartjs-adapter-luxon"
+
+type GameNumberChartProps = {
+  labels: string[]
+  data: number[]
+}
+
+const longNumberFormatter = new Intl.NumberFormat("en-us")
+
+export function GameNumberChart(props: GameNumberChartProps) {
+  let canvas: HTMLCanvasElement
+  onMount(() => {
+    Chart.register(Title, Tooltip, Colors, TimeScale)
+  })
+
+  const chartData: ChartData = {
+    labels: props.labels,
+    datasets: [
+      {
+        label: "Games",
+        data: props.data,
+        backgroundColor: "#5EC269",
+      },
+    ],
+  }
+
+  const chartOptions: ChartOptions = {
+    responsive: true,
+    maintainAspectRatio: true,
+    animation: false,
+    interaction: {
+      intersect: false,
+      mode: "index",
+    },
+    scales: {
+      x: {
+        type: "time",
+        time: { unit: "day", tooltipFormat: "DD" },
```

Comment:
I don't think this library is a great addition. It uses a global dependency that registers itself of Chart outside of modern coding patterns, and also assumes people know of this plugin and the syntax. Luxon is literally a wrapper around `Intl.DateFormat` and while it has some utility around calculations that's not what it is being used here for.